app/models/disc_evidence_coversheet.rb

```diff
@@ -34,7 +34,7 @@ def current_date
   end
 
   def fee_scheme
-    @fee_scheme || claim.fee_scheme.name.upcase
+    @fee_scheme || defined?(claim.fee_scheme.name)
```

Comment:
you can just do a claim&.fee_scheme&.name here.

i saw this sentry error but I assumed it was an abberation as every claim should have a fee scheme and every fee scheme has a name.

but this would prevent that error in such circumstances